src/jdk.jpackage/windows/classes/jdk/jpackage/internal/resources/MsiInstallerStrings_ja.wxl

```diff
@@ -1,5 +1,5 @@
 <?xml version = '1.0' encoding = 'utf-8'?>
-<WixLocalization Culture="ja-jp" xmlns="http://schemas.microsoft.com/wix/2006/localization" Codepage="932">

```

Comment:
Please revert the value of the `Culture` attribute.

Reply:
Resolved in recent commit